examples/basics/data_rows.ipynb

```diff
@@ -2,18 +2,22 @@
   "cells": [
```

Comment:
<p>Could you use the simplify version of the imports import (eg. "import labelbox as lb") and modify method calls throughout the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/875/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>